.stylelintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "stylelint-config-standard-scss",
+  "rules": {
+    "selector-class-pattern": "^[a-z0-9]+(-[a-z0-9]+)*$",
+    "string-quotes": "single"
+  },
+  "ignoreFiles": [
+    "dist/*"
+  ]
+}
```

Comment:
[EOF](https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log)

https://eslint.org/docs/rules/eol-last